l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -12,7 +12,9 @@
 import string
 import random
 
+# pylint: disable=missing-import-error
```

Comment:
Qui ci sarebbe da aggiungere il `try..except` come indicato in https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#importerror

Reply:
fatto